docs/private-dns/api/reference.md

```diff
@@ -10,14 +10,99 @@ toc_max_heading_level: 4
     https://api.adguard-dns.io/static/swagger/openapi.json to markdown using
     https://swagger-markdown-ui.netlify.app/.
 
+    Changelog is from here:
+    https://api.adguard-dns.io/static/api/CHANGELOG.md
+
     If you want to change it, ask the developers to change the OpenAPI spec.
 -->
 
 ## AdGuard DNS API
 
 DNS API documentation
 
-## Version: 1.4
+## AdGuard DNS API Change Log
+
+### v1.0
```

Comment:
One could argue versions should go the other way around, from most recent to the oldest one

Reply:
It's also auto-generated